devices.js

```diff
@@ -4713,6 +4713,66 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['RM01'],
+        model: 'RM01',
```

Comment:
Can you change the model to `6735/6736/6737` (this is the same as how we handle other devices sharing the same `zigbeeModel`.

Reply:
done